src/databases/Tecplot/TecplotFile.C

```diff
@@ -2211,6 +2211,7 @@ TecplotDataRecord::CalculateOffsets(const TecplotZone &zone)
     else
     {
         int numNodes = zone.GetNumNodes();
+        int numElements = zone.GetNumElements();
```

Comment:
Would you mind please adding a modification comment, which includes your name/GitHub handle in the function pre-amble?